Mathlib/Tactic/Linter/TextBased.lean

```diff
@@ -391,6 +391,8 @@ register_option linter.modulesForbiddenWindows : Bool := { defValue := true }
 COM6, COM7, COM8, COM9, COM¹, COM², COM³, LPT1, LPT2, LPT3, LPT4, LPT5, LPT6, LPT7, LPT8, LPT9,
 LPT¹, LPT² or LPT³ in its filename, as these are forbidden on Windows.
 
+Also verify that module names contain no forbidden characters such as `*` or `!`.
+
 Source: https://learn.microsoft.com/en-gb/windows/win32/fileio/naming-a-file.
 Return the number of module names violating this rule. -/
 def modulesForbiddenWindows (opts : LinterOptions) (modules : Array Lean.Name) : IO Nat := do
```

Comment:
```suggestion
def modulesOSForbidden (opts : LinterOptions) (modules : Array Lean.Name) : IO Nat := do
```
since it's not Windows-specific anymore?